### PR TITLE
Modal use item out combat

### PR DESCRIPTION
**Pull request by andres123dbh**
- Using items on loomies that are not busy
- Update item inventory view
- If don't have loomies it shows a message
- When use the item successfully, stay in the item inventory view.

### CODE REVIEW

File: src/components/Modals/UseItemOutCombat.tsx [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+}
+
+export const UseItemOutCombatModal = ({
+  selectedItem,
+  isVisible,
+  toggleVisibilityCallback,
+  closeModalItem
+}: IProps) => {
+  const { showErrorToast, showSuccessToast } = useToastAlert();
+  const [team, setTeam] = useState<string>();
```
Comment (PedroChaparro) [inline src/components/Modals/UseItemOutCombat.tsx:27]: What is this state. Seems it isn't a team, you are assigned a Loomie Id as the value.

**Review by PedroChaparro — CHANGES_REQUESTED** [review]
Please, don't push an unique huge commit with all your changes, that makes it harder to review the PR. You can push small/medium commits for each "feature", for example: 

- Commit 1: Create the modal 
- Commit 2: Show the modal and send a request to the backend
- Commit 3: Update the inventory view

File: src/components/Modals/UseItemOutCombat.tsx (hunk 1 above)
Comment (andres123dbh) [reply src/components/Modals/UseItemOutCombat.tsx:27]: I forgot to change the name, it's to select a loomie to which to apply the item

**Review by PedroChaparro — APPROVED** [review]

**Review by Woynert — APPROVED** [review]
:+1:

**Pull request merged**